src/js/themes/base.js

```diff
@@ -328,6 +328,26 @@ export const generate = (baseSpacing = 24, scale = 6) => {
       // extend: undefined,
     },
     button: {
+      size: {
+        small: {
+          ...fontSizing(-1),
+          border: {
+            radius: `${baseSpacing * 0.7}px`,
```

Comment:
We should truncate to be whole pixel values.

Reply:
To be honest, I was just putting placeholder's values for the radius, except the default 'medium' that behaves as it used to. 
If we will proceed with this direction I would ask @L0ZZI or @gregfuruiye for the border.radius accurate values.

Reply:
sounds good